### PR TITLE
SOFB.ENH: Add PerformanceWindow.

### PR DESCRIPTION
**Pull request by fernandohds564**

### CODE REVIEW

File: pyqt-apps/siriushla/as_ap_sofb/ioc_control/main.py [hunk 1]
```diff
@@ -444,3 +458,93 @@ def __init__(self, parent, prefix, ctrls, acc='SI'):
...
+        lay_iters.addWidget(ld_tout, 1, 2, alignment=Qt.AlignCenter)
+        lay_iters.addWidget(ld_diff, 1, 3, alignment=Qt.AlignCenter)
+
+        lbl_psd = QLabel('# of PSs with Diffs:', self)
+        lbl_avg = QLabel('AVG', self, alignment=Qt.AlignCenter)
+        lbl_std = QLabel('STD', self, alignment=Qt.AlignCenter)
+        lbl_max = QLabel('MAX', self, alignment=Qt.AlignCenter)
+        ld_avg = SiriusLabel(self, lamb('LoopPerfDiffNrPSMax-Mon'))
+        ld_std = SiriusLabel(self, lamb('LoopPerfDiffNrPSAvg-Mon'))
+        ld_max = SiriusLabel(self, lamb('LoopPerfDiffNrPSStd-Mon'))
```
Comment (anacso17) [inline pyqt-apps/siriushla/as_ap_sofb/ioc_control/main.py:501]: I think the pvnames here are mixed

Comment (fernandohds564) [reply pyqt-apps/siriushla/as_ap_sofb/ioc_control/main.py:501]: right! thanks.

File: pyqt-apps/siriushla/as_ap_sofb/ioc_control/main.py [hunk 2]
```diff
@@ -444,3 +457,94 @@ def __init__(self, parent, prefix, ctrls, acc='SI'):
...
+
+        labs = [
+            'Get Orbit', 'Get Kick', 'Calculate', 'Process', 'Apply', 'Total']
+        nms = ['GetO', 'GetK', 'Calc', 'Proc', 'App', 'Tot']
+        for i, (nm, lab) in enumerate(zip(nms, labs)):
+            lbl = QLabel(lab, self)
+            lay_tim.addWidget(lbl, i+2, 0)
+            for j, ld in enumerate(['Avg', 'Std', 'Min', 'Max']):
+                ld_ = SiriusLabel(
+                    self, lamb(f'LoopPerfTim{lab:s}{ld:s}-Mon'))
```
Comment (anacso17) [inline pyqt-apps/siriushla/as_ap_sofb/ioc_control/main.py:537]: `labs` have some strings with spaces, I think the spaces have to be removed here

File: pyqt-apps/siriushla/as_ap_sofb/ioc_control/main.py (hunk 1 above)
Comment (fernandohds564) [reply pyqt-apps/siriushla/as_ap_sofb/ioc_control/main.py]: done!

File: pyqt-apps/siriushla/as_ap_sofb/ioc_control/main.py (hunk 2 above)
Comment (fernandohds564) [reply pyqt-apps/siriushla/as_ap_sofb/ioc_control/main.py:537]: right! i was supposed to be `nm` not `lab` there! Thanks!

Comment (fernandohds564) [reply pyqt-apps/siriushla/as_ap_sofb/ioc_control/main.py]: done!

**Review by anacso17 — APPROVED** [review]

**Review by murilobalves — APPROVED** [review]

**Review by xresende — APPROVED** [review]

**Pull request merged**